std/experimental/allocator/mallocator.d

```diff
@@ -375,6 +375,7 @@ version(Posix)
     AlignedMallocator.instance.alignedReallocate(c, 32, 32);
     assert(c.ptr);
 
+    version (DragonFlyBSD) {} else                    /* FIXME: assertion below fails, have not been able to figure out why, yet */
     assert(!AlignedMallocator.instance.alignedReallocate(c, size_t.max, 4096));
```

Comment:
If this assert fails it follows that `alignedReallocate` successfully called `posix_memalign`, allocating more virtual memory than what could be reasonably available on the system. Can you try to debug this test and see what exactly breaks down?

Reply:
Reported this error to dragonfly ([dragonfly bug report](https://bugs.dragonflybsd.org/issues/3114))
See [PR Discussion](https://github.com/dlang/druntime/pull/1999#discussion_r157536030)
Will update this entry to add both these links, so the issue can be tracked.

Reply:
Thanks!